.github/CHANGELOG.md

```diff
@@ -50,6 +50,38 @@
 
 <h3>Improvements</h3>
 
+* The ``VQECost`` class now provides observable optimization using the ``optimize`` argument,
```

Comment:
Did we also decide to rename `VQECost`? Probably makes sense to do that now, especially if it will be present in the changelog.

We can retain `VQECost` as an alias, but have it raise a deprecation warning.

Reply:
Cool, happy for this to be done in a new PR!